notebooks/lesson02_exploratory-data-analysis.ipynb

```diff
@@ -0,0 +1,2971 @@
+{
```

Comment:
<p>I'd add a remark about the choice of axis since it happened countless times to me that I dropped columns instead of rows.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>Good catch - i'll add something </p>